crates/matrix-sdk/src/authentication.rs

```diff
@@ -80,6 +80,15 @@ impl AuthSession {
         }
     }
 
+    /// Take the matrix user information of this session.
+    pub fn take_meta(self) -> SessionMeta {
```

Comment:
The `take_` prefix usually implies a `&mut` method. I think this should probably be called `into_meta` instead.

Reply:
Ah, that sounds more natural indeed, thank you.